src/components/QExpansionPanel/QExpansionPanels.vue

```diff
@@ -0,0 +1,10 @@
+<template>
```

Comment:
Why this component isn't removed ?

Reply:
As mentionned in the PR it was for future improvements but fine I'm removing it